lib/plurimath/math/function/table.rb

```diff
@@ -30,8 +30,8 @@ def to_mathml_without_math_tag
               column_lines << "none"
             end
           end
-          columnlines_attrs = "columnlines=\"#{column_lines.join(' ')}\"" if column_lines.include?("solid")
-          table_tag = "<mtable #{columnlines_attrs}>#{table_value}</mtable>"
+          columnlines_attrs = " columnlines=\"#{column_lines.join(' ')}\"" if column_lines.include?("solid")
+          table_tag = "<mtable#{columnlines_attrs}>#{table_value}</mtable>"
```

Comment:
Will these go away by using Ox to manipulate nodes?

Reply:
Yes, these will go away in the reverse conversion using Ox Parser. I’m currently working on it.